config/secrets.yml

```diff
@@ -120,7 +120,7 @@ test:
   api_promo_base_uri: "" # http://127.0.0.1:8194
   active_promo_id: "free-bats-2018q1"
   api_auth_token: "fake_api_auth_token"
-  api_eyeshade_base_uri: http://127.0.0.1:8194
+  api_eyeshade_base_uri: http://127.0.0.1:3001
```

Comment:
yep, not sure how that port got switched.  I left it as it's own commit.

this really won't matter though since eyeshade can really only be run with docker now (all other install instructions have been removed) https://github.com/brave-intl/bat-ledger